docs/releases/1.1.0.rst

```diff
@@ -15,6 +15,9 @@ Major Changes
   work but will trigger a deprecation warning, and ``channels.tests`` will be
   removed completely in version 1.3.
 
+* Add ``ChannelLiveServerTestCase`` for integration testing.  It is
+  possible to run Selenium with JavaScript which operates WebSockets.
```

Comment:
This now needs removing from these release notes as we're further along; don't worry about adding release notes, I'll do that when I make the release!